src/makeCalculator.js

```diff
@@ -4,7 +4,28 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (num) => calculator.result + num,
+    subtract: (num) => calculator.result - num,
```

Comment:
[CODE STYLE] It's recommended to use arithmetic operators with assignment operator `=`. So, instead of `calculator.result - num`, you should write `calculator.result -= num`.